boards/st/stm32l562e_dk/Kconfig.defconfig

```
@@ -25,4 +25,9 @@ config BT_HCI_VS
 
 endif # BT
 
+if MIPI_DBI_STM32_FMC
+	config MIPI_DBI_STM32_FMC_MEM_BARRIER
+	default n
+endif # MIPI_DBI_STM32_FMC
```

Nit: `config XXX` indentation not required. Also for one symbol it could be put in a more concise way
```suggestion
config MIPI_DBI_STM32_FMC_MEM_BARRIER
	default n if MIPI_DBI_STM32_FMC
```